docs/kernel-policy.md

```diff
@@ -21,10 +23,10 @@ Once a kernel version is officially enabled, it is supported for a **minimum of
 2 years**. Adding support for a new kernel version will result in a Firecracker
 release only if compatibility changes are required.
 
-| Host kernel | Guest kernel v4.14 | Guest kernel v5.10 | Min. end of support |
-| ----------: | :----------------: | :----------------: | ------------------: |
-|       v5.10 |         Y          |         Y          |          2024-01-31 |
-|        v6.1 |         Y          |         Y          |          2025-10-12 |
+| Host kernel | Guest kernel v4.14 (deprecated) | Guest kernel v5.10 | Guest kernel v6.1 | Min. end of support |
```

Comment:
nitnit: should we put the deprecated in the column below next to Y?
```
|       v5.10 |      Y - deprecated      |         Y          |         Y         |          2024-01-31 |
```

Reply:
done